src/components/KYCWall/types.ts

```diff
@@ -0,0 +1,71 @@
+import {ForwardedRef, SyntheticEvent} from 'react';
+import {NativeTouchEvent} from 'react-native';
+import {OnyxEntry} from 'react-native-onyx';
+import {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+import {Route} from '@src/ROUTES';
+import {Report} from '@src/types/onyx';
+
+type Source = ValueOf<typeof CONST.KYC_WALL_SOURCE>;
+
+type TransferMethod = ValueOf<typeof CONST.WALLET.TRANSFER_METHOD_TYPE>;
+
+type DOMRectProperties = 'top' | 'bottom' | 'left' | 'right' | 'height' | 'x' | 'y';
+
+type DomRect = Pick<DOMRect, DOMRectProperties>;
```

Comment:
Why do we need to pick those properties? Can't we just use `DOMRect`?

Reply:
I'm using this custom DomRect type to match with what comes from the getClicketTargetLocation function from the libs, this function returns a similar type.